PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -102,7 +103,7 @@ function transformReports(inputReportFiles: string[], index: number)
         const saxStream = sax.createStream(true, {});
         saxStream.on("opentag", function (node) {
             if (node.name == 'StdViols') {
-                if (!bLegacyReport){
+                if (!bLegacyReport || bCPPProReport) {
```

Comment:
This whole if condition can be removed because I think it was there to prevent handling cpp professional report but we are adding support now.

Reply:
I guess in this case we don't need "bCPPProReport" anymore because you already generate the map for rule id and analyzer id in earlier task which can be used to get rule doc.